libs/api/libbpf_program.cpp

```diff
@@ -293,7 +293,8 @@ bpf_program__get_type(const struct bpf_program* program)
 void
 bpf_program__set_type(struct bpf_program* program, enum bpf_prog_type type)
 {
-    program->program_type = *_get_ebpf_program_type(type);
+    ebpf_program_type_t* program_type = _get_ebpf_program_type(type);
+    program->program_type = (program_type != nullptr) * program_type : EBPF_PROGRAM_TYPE_UNSPECIFIED;
```

Comment:
Should it be 

"program->program_type = (program_type != nullptr) `?` * program_type : EBPF_PROGRAM_TYPE_UNSPECIFIED;"

Reply:
already fixed